compiler/rustc_ast/README.md

```diff
@@ -1,6 +1,5 @@
 The `rustc_ast` crate contains those things concerned purely with syntax
-– that is, the AST ("abstract syntax tree"), parser, pretty-printer,
-lexer, macro expander, and utilities for traversing ASTs.
+– that is, the AST ("abstract syntax tree"), along with some definitions for tokens and token streams, data structures/traits for mutating ASTs, and shared definitions for other AST-related parts of the compiler (like the lexer and macro-expansion).
```

Comment:
Hmm, I'm not even sure why this file exists. I thought all the `compiler/` READMEs were moved to the rustc-dev-guide.

Reply:
I'd assume that's because this gets published to crates.io as part of the auto publish process

Reply:
Ah, I see 👍